lua/luasnip/extras/postfix.lua

```diff
@@ -65,17 +72,16 @@ local function postfix(context, nodes, opts)
 		user_callback = { user_callback, { "nil", "function" } },
 	})
 
-	local match_pattern
-	if type(context) == "string" then
-		context = {
-			trig = context,
-		}
-		match_pattern = matches.default
-	else
-		match_pattern = context.match_pattern or matches.default
-	end
+	local match_pattern = context.match_pattern or matches.default
```

Comment:
WDYT about doing this check after `wrap_context` (btw, good that you replaced that :+1:)?
Feels a bit weird to index a potential string, I know it doesn't error, but doesn't sit quite right :sweat_smile: